python/ray/services.py

```diff
@@ -76,11 +83,11 @@ def kill_process(p):
     os.kill(p.pid, signal.SIGINT) # Give process signal to write profiler data.
     time.sleep(0.1) # Wait for profiling data to be written.
   p.kill()
-  time.sleep(0.05) # is this necessary?
+  time.sleep(0.01) # is this necessary?
```

Comment:
have you tried it with 0.0 ? Unless the time module is special-casing zero, this should result in a usleep syscall, which would yield the core.